tools/cldr-apps/src/test/java/org/unicode/cldr/unittest/web/TestMisc.java

```diff
@@ -96,18 +91,13 @@ public void TestGitHash() {
                 errln("❌ getCldrBaseDirHash is shorter than 8 chars: " + hash);
             }
         }
-        final String strings[] = {"CLDR-Apps", "CLDR-Tools"};
```

Comment:
again, i dont' think ther'es a reason to remove this functionality entirely.

Reply:
we shouldn't keep dead/broken code around just based on wishful thinking -- there's too much of that already -- so it should be decided whether we're going to implement a cldr-apps war hash during v42 -- the return on investment seems dubious

as a compromise, we could have a brief comment indicating that a cldr-apps war hash could be implemented similar to the cldr-code jar hash

Reply:
It's already out so I'm fine to keep it so. It's more of a bug than wishful thinking though!